Earthfile

```diff
@@ -367,79 +348,158 @@ uki-tools-image:
 # NOTE: NEVER EVER EVER use this keys for signing anything that its going outside your computer
 # This is for easy testing SecureBoot locally for development purposes
 # Installing this keys in other place than a VM for testing SecureBoot is irresponsible
-uki:
```

Comment:
I'm also making some changes on this. We may get some conflicts: https://github.com/kairos-io/kairos/pull/2081/files#diff-2705f6016a64189abd14041e2f3e8dbb829553295d9088e4eb42eccb5e61bf47R367